subsys/net/lib/http/Kconfig

```diff
@@ -65,6 +65,14 @@ config HTTP_PARSER
 	This parser requires some string-related routines commonly
 	provided by a libc implementation.
 
+config HTTP_PARSER_URL_ONLY
+	bool "HTTP URL parser only support"
+	default n
+	select HTTP_PARSER
+	help
+	This option enables URL parser part of the http_parser
```

Comment:
Perhaps just say here "This option only enables URL parser of the http_parser library."